FreshRSS/FreshRSS
# Fixed: up.svg in Ansum/Mapco theme

Regression from #4788

Themes: Mapco & Ansum

Before:
The arrow icon is bigger in the open state than in closed state.
![grafik](https://user-images.githubusercontent.com/1645099/206908604-9cad478a-9038-4f50-b8f7-a2c375603ad0.png)

![defect](https://user-images.githubusercontent.com/1645099/206908902-2372b955-48ec-445b-8a7e-7c80f954fe56.gif)

After:
![fixed](https://user-images.githubusercontent.com/1645099/206908949-c12f88a0-4a82-473e-922c-b04c240790c9.gif)


Changes proposed in this pull request:

- up.svg of Ansum and Mapco theme (size before: 10x16, after: 16x16)


How to test the feature manually:

1. use Ansum or Mapco theme
2. open/close a category in normal view
3. see the arrow icon

Pull request checklist:
